dev/integration_tests/ios_platform_view_tests/ios/PlatformViewUITests/PlatformViewUITests.m

```diff
@@ -6,14 +6,26 @@
 
 static const CGFloat kStandardTimeOut = 60.0;
 
-@interface XCUIElement(KeyboardFocus)
+@interface XCUIElement(Test)
 @property (nonatomic, readonly) BOOL flt_hasKeyboardFocus;
+- (void)flt_forceTap;
 @end
 
-@implementation XCUIElement(KeyboardFocus)
+@implementation XCUIElement(Test)
 - (BOOL)flt_hasKeyboardFocus {
   return [[self valueForKey:@"hasKeyboardFocus"] boolValue];
 }
+
+- (void)flt_forceTap {
+  if (self.isHittable) {
+    [self tap];
```

Comment:
So it needs to be hittable for `tap` on the element to work?

Reply:
yeah otherwise it will just crash.